### PR TITLE
fix: update link to `vercel/ai` github repo

### PR DESCRIPTION
**Pull request by styfle**
`vercel-labs/ai` => `vercel/ai`

### CODE REVIEW

**Review by jaredpalmer — APPROVED** [review]

File: packages/core/package.json [hunk 1]
```diff
@@ -118,13 +118,13 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "https://github.com/vercel-labs/ai#readme",
+  "homepage": "https://vercel.com/templates/ai",
```
Comment (styfle) [inline packages/core/package.json:121]: @jaredpalmer Thoughts on the homepage here?

Comment (jaredpalmer) [reply packages/core/package.json:121]:
```suggestion
  "homepage": "https://sdk.vercel.ai/docs",
```

Comment (styfle) [reply packages/core/package.json:121]: I'm going to change it to https://sdk.vercel.ai/docs which feels more appropriate

Comment (styfle) [reply packages/core/package.json]: Glad we both came to the same conclusion 😄

**Pull request merged**